internal/inc/file_internal.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FILE_INTERNAL_H
+#define FILE_INTERNAL_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdint.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"

```

Comment:
there are no mockable_function here...

Reply:
removed in next iteration

---
In reply to: [450466885](https://github.com/Azure/azure-c-pal/pull/7#discussion_r450466885) [](ancestors = 450466885)